frontend/catalyst/cuda/__init__.py

```diff
@@ -24,8 +24,56 @@
 from catalyst.cuda.catalyst_to_cuda_interpreter import interpret
 
 
-def qjit(fn=None, **kwargs):
-    """Wrapper around QJIT for CUDA-quantum."""
+def cudaqjit(fn=None, **kwargs):
```

Comment:
Yes, the issue was Sphinx :( atm we are using
```rest
:func:`~.qjit`
```
to automatically cross reference the QJIT docstring in our docs; having two functions called `qjit` was breaking this. The other option was to update the docs to use
```rest
:func:`~catalyst.qjit` and :func:`~.cuda.qjit`
```
everywhere, but it would have led to a big PR, and also issues in the PL documentation that would have needed fixing.

Reply:
I agree with you though, I prefer `cuda.qjit` personally. I'm open to changing this back when we have more time

Reply:
Sounds good 👍